tutor/templates/config/defaults.yml

```diff
@@ -22,7 +22,7 @@ DOCKER_IMAGE_MONGODB: "docker.io/mongo:4.4.25"
 DOCKER_IMAGE_MYSQL: "docker.io/mysql:8.1.0"
 DOCKER_IMAGE_PERMISSIONS: "{{ DOCKER_REGISTRY }}overhangio/openedx-permissions:{{ TUTOR_VERSION }}"
 # https://hub.docker.com/_/redis/tags
-DOCKER_IMAGE_REDIS: "docker.io/redis:7.2.1"
+DOCKER_IMAGE_REDIS: "docker.io/redis:7.2.4"
```

Comment:
You should also update the default value of `DOCKER_IMAGE_REDIS` in the tutor docs. (and this value is outdated by the way)